### PR TITLE
Feature/iteration 4 and beyond

### PR DESCRIPTION
**Pull request by Billwilke42**
### What's this PR do?
Adds screenshots to the readME and fixed the bug in the delete mini posters function.
### Where should the reviewer start?
Line 222
### How should this be manually tested?
Open in browser, and save some posters, double click the poster to make it delete.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,10 +36,10 @@ _Hint: go check out the HTML and CSS files to see how the form and saved posters
 ## Iteration 2 - Creating a New Poster
 
 Form being filled out:
-![screenshot of form](/readme-imgs/form.png)
+![Screen-Shot-2020-04-01-at-12-51-59-PM.png](https://i.postimg.cc/d0nbV1hr/Screen-Shot-2020-04-01-at-12-51-59-PM.png)
 
 Once poster is saved:
-![screenshot of result](/readme-imgs/form-result.png)
+![Screen-Shot-2020-04-01-at-12-52-14-PM.png](https://i.postimg.cc/sXNb5N92/Screen-Shot-2020-04-01-at-12-52-14-PM.png)
```
Comment (jdvolk) [inline README.md:42]: nice screen shot

File: README.md [hunk 2]
```diff
@@ -14,17 +14,17 @@ A boilerplate repo.
 
 ### Iteration 0 - Main Page
 
-![screenshot of main page showing poster](/readme-imgs/homepage.png)
+![Screen-Shot-2020-04-01-at-2-52-36-PM.png](https://i.postimg.cc/dV78HSYC/Screen-Shot-2020-04-01-at-2-52-36-PM.png)
```
Comment (jdvolk) [inline README.md:17]: looks good

File: README.md (hunk 1 above)
Comment (Billwilke42) [reply README.md:42]: Thanks!

File: src/main.js [hunk 3]
```diff
@@ -219,10 +219,10 @@ function createPosterHtml(posterObj) {
   </div>`;
 }
 
-function deleteMiniPoster() {
+function deleteMiniPoster(event) {
   var deletedPosters = [];
   var mini = document.querySelector(".mini-poster");
-  var closestElement = mini.closest("article > div");
+  var closestElement = event.target.closest('.mini-poster')
```
Comment (jdvolk) [inline src/main.js:225]: perfect

**Pull request merged**